docs/setup.md

```diff
@@ -62,19 +62,29 @@ The following permission allow a user to interact with the **FortiSOAR AI Assist
 >[!Note]
 > The **FortiSOAR AI Assistant** bot becomes available to interact on modules that contain records like Alerts, Indicators, or Incidents. The bot is still available to interact on all such modules even if there are no records present.
 
+## Configuring Voice Support
+
+**Microphone Setup**: Confirm that a functional microphone is connected and enabled. The browser must have permissions to access the microphone.
+
+### Best Practices for Voice Commands
+
+- **Speak Clearly**: Enunciate clearly and at a moderate pace.
+- **One Command at a Time**: Issue one command at a time to ensure accurate processing.
+- **Pause Between Commands**: Allow brief pauses to enable FortiAI to process each command.
+
 # Configuration
 
 The **FortiAI** solution depends on the following connectors and widgets:
 
 - **AI Assistant Utils** connector to help interact with LLMs like OpenAI.
 
 >[!NOTE]
->The AI Assistant Utils connector does not require a configuration. For more details, see the [AI Assistant Utils Connector](https://docs.fortinet.com/document/fortisoar/1.0.0/ai-assistant-utils/690/ai-assistant-utils-v1-0-0) document.
+>The AI Assistant Utils connector does not require a configuration. For more details, see the [AI Assistant Utils Connector](https://docs.fortinet.com/fortisoar/connectors/ai-assitant-utils) document.
 
-- The **AI Assistant** widget to add the **FortiSOAR AI Assistant** bot to the FortiSOAR user interface. For more details, see the [AI Assistant Widget](https://docs.fortinet.com/document/fortisoar/1.0.0/ai-assistant-utils/690/ai-assistant-utils-v1-0-0) document.
+- The **AI Assistant** widget to add the **FortiSOAR AI Assistant** bot to the FortiSOAR user interface. For more details, see the [AI Assistant Widget](https://docs.fortinet.com/fortisoar/connectors/ai-assitant-utils) document.
```

Comment:
To AI Assistant widget doc link

Reply:
@cs-abhishek-shukla

Reply:
Pointing to Dev repo as the branch is merged in develop after release, ok? @cs-suyog-jinturkar @cs-ankit-gupta